From 6935bd0821395051c0483b0ee393d2d4567f6f0c Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 24 May 2017 12:15:20 -0400
Subject: [PATCH] Ticket 49241 - add symblic link location to db2bak.pl output

Description:  If a symbolic link is used for the script's backup
              location then add info to the output.

https://pagure.io/389-ds-base/issue/49241

Reviewed by: firstyear(Thanks!)

(cherry picked from commit 95a7f23262076d90fdc8a9ec76e131e9e4c09bcc)
---
 ldap/admin/src/scripts/db2bak.pl.in | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/db2bak.pl.in b/ldap/admin/src/scripts/db2bak.pl.in
index 335285e..352a01e 100644
--- a/ldap/admin/src/scripts/db2bak.pl.in
+++ b/ldap/admin/src/scripts/db2bak.pl.in
@@ -105,7 +105,12 @@ if ($archivedir eq "") {
     } else {
         $symname = $archivedir;
     }
-    print("Back up directory: $archivedir\n");
+    if ($symname eq "") {
+        print("Back up directory: $archivedir\n");
+    } else {
+        print("Back up directory: $archivedir -> $mybakdir/$archivebase\n");
+    }
+
     # If an archive dir is specified, create it as a symlink pointing
     # to the default backup dir not to violate the selinux policy.
     $archivedir = "${mybakdir}/${archivebase}";
-- 
2.9.4

